### ISSUE TITLE
AttachmentOptions missed logic

### ISSUE THREAD

**Issue by wontem** [greggman/twgl.js#168]
During the investigation, I found that parameter `AttachmentOptions['attach']` is never used, but fully documented and has typescript declaration:
https://github.com/greggman/twgl.js/blob/9ed693912454c5b041776683f3b3633df7e6ca7e/src/framebuffers.js#L88
https://github.com/greggman/twgl.js/blob/9ed693912454c5b041776683f3b3633df7e6ca7e/npm/base/index.d.ts#L22

The actual attachment point is being selected here:
https://github.com/greggman/twgl.js/blob/9ed693912454c5b041776683f3b3633df7e6ca7e/src/framebuffers.js#L200

I'll be glad to create a pull request, but I confused with the idea of this parameter. Is it redundant legacy code and should be removed, or is it an option, that should be used as an attachment point if defined?

**Comment by munrocket** [comment]
Yeah, looks like an outdated comment that become a property in typescript.

Here usage of attachments
https://github.com/greggman/twgl.js/blob/9ed693912454c5b041776683f3b3633df7e6ca7e/examples/mulitple-render-targets-3d-texture.html#L169

**Comment by wontem** [comment]
So the order is important. Maybe it's better to have the option of using a specific attachment point? Or, at least, add attachment point into output object? It would be great to have the availability of reusing it in some cases. For `gl.drawBuffers` for example
https://github.com/greggman/twgl.js/blob/9ed693912454c5b041776683f3b3633df7e6ca7e/examples/mulitple-render-targets-3d-texture.html#L175
So I mean that it's not transparent logic right now. The other option is to add some parameter/s, which will run `gl.drawBuffers` under the hood

**Comment by greggman** [comment]
Good catch. I'm not sure off top of my head what the best solution is. twgl is not trying to cover all of WebGL. It's just trying to make some things less verbose.

I don't think generating an input for drawBuffers is real all that useful. It's a really tiny API. Just an array with 4-8 constants. Does that really need an helper? 

The simplest thing would be to remove it. Clearly no one is using it if it never worked. .

Otherwise the intent was

```
let attachmentPoint = attachmentOptions.attach || getAttachmentPointForFormat(format);
```
